.github/deploy_docs.sh

```diff
@@ -134,13 +136,16 @@ cp -R $SOURCE_DIR/* $DEST_DIR/
 echo "Staging files in git"
 git add $DEST_DIR/
 
+echo "Preparing to deploy documentation"
+
 if [[ -z $(git status --porcelain) ]]; then
     echo "Nothing has changed, nothing needs pushing."
 else
     echo "Making commit of new files"
     git config user.email "sphinx@example.org"
     git config user.name "Sphinx"
-    git commit -m "Automated update ${TRAVIS_COMMIT:-}"
+    COMMIT_HASH=$(git rev-parse HEAD)
+    git commit -m "Automated update ${COMMIT_HASH}"
```

Comment:
Sadly that will give the most recent git commit on the documentation repository, e.g. the commit comment for https://github.com/erik-whiting/docs/commit/ba92d210963f117487d2088d3dcb619e174701d2 points to https://github.com/erik-whiting/docs/commit/b6d8aef0eaead6759ef239dcb057ebf4ddb117ba

You can probably move the ``COMMIT_HASH=$(git rev-parse HEAD)`` line up the script before leaving the clone of the source code repository?